.github/workflows/release.yml

```diff
@@ -182,6 +197,20 @@ jobs:
                   echo "$GPG_PASSPHRASE" | gpg --passphrase-fd 0 --pinentry-mode loopback --batch -ab lighthouse-${{ needs.extract-version.outputs.VERSION }}-${{ matrix.arch }}.tar.gz
                   mv *tar.gz* ..
 
+            - name: Configure GPG and create artifacts Windows
+              if: startsWith(matrix.arch, 'x86_64-windows')
+              env:
+                  GPG_SIGNING_KEY: ${{ secrets.GPG_SIGNING_KEY }}
+                  GPG_PASSPHRASE: ${{ secrets.GPG_PASSPHRASE }}
+              run: |
+                  echo $env:GPG_SIGNING_KEY | gpg --batch --import
+                  mkdir artifacts
+                  move $env:USERPROFILE/.cargo/bin/lighthouse.exe ./artifacts
+                  cd artifacts
+                  tar -czf lighthouse-${{ needs.extract-version.outputs.VERSION }}-${{ matrix.arch }}.tar.gz lighthouse.exe
+                  gpg --passphrase "$env:GPG_PASSPHRASE" --batch --pinentry-mode loopback -ab lighthouse-${{ needs.extract-version.outputs.VERSION }}-${{ matrix.arch }}.tar.gz
```

Comment:
I couldn't get this passphrase working with the `echo $PASSPHRASE | gpg ...` pattern, not sure if this always happens on Windows with gpg, or just in Github Actions. I know that pattern is recommended on linux systems to keep the password out of the shell history, but not sure if that is applicable on windows still?

Would appreciate it if someone with a stronger sense of Windows security could comment on if this is ok.

Reply:
I think the shell history thing shouldn't matter because the VMs are ephemeral, and Github scrubs secrets from the logs.

The [Github docs](https://docs.github.com/en/actions/reference/encrypted-secrets#using-encrypted-secrets-in-a-workflow) do mention keeping the secrets out of the CLI args so that they're not visible to auditing processes/other users – although I hope there are no other users on our CI VMs!

I don't see how `echo $secret | gpg` helps, as the secret is still a CLI argument to `echo`. I think we could use PowerShell's `Get-Item` though:

```
Get-Item Env:GPG_PASSPHRASE | gpg --passphrase-fd 0 --batch --pinentry-mode loopback -ab .\lighthouse.exe
```

That seems to work on my machine. If the CI runner has issues with `--passphrase-fd` we could use an intermediate file and `--passphrase-file`, or just leave it as-is. I think our Linux runner suffers from the same issue with `echo`.

Reply:
Yea, there were still issues with `Get-Item` so I had to revert that. I'll just leave it as the `--passphrase` flag for now